plugins/catalog/report.api.md

```diff
@@ -347,6 +347,20 @@ export interface DependsOnResourcesCardProps {
 // @public
 export const EntityAboutCard: (props: AboutCardProps) => JSX.Element;
 
+// @public (undocumented)
+export function EntityContextMenu(props: {
+  UNSTABLE_extraContextMenuItems?: Array<{
+    title: string;
+    Icon: IconComponent;
+    onClick: () => void;
+  }>;
+  UNSTABLE_contextMenuOptions?: {
+    disableUnregister: boolean | 'visible' | 'hidden' | 'disable';
+  };
+  onUnregisterEntity: () => void;
+  onInspectEntity: () => void;
```

Comment:
Sorry didn't spot these in the framework SIG, the fact that these aren't baked into the context menu is a bit of an issue imo. I think we're still at the state where you're better off duplicating this code for now, or alternatively using yarn patches if the changes you want to make are small

Reply:
No worries, and thank you again for taking a look at this yesterday.

Sorry if I'm missing the obvious here, but could you help me understand what the concern is in particular with having these props available?

Reply:
Yeah, I also do not understand why having these props available is a problem. They seem like a great idea to me.